### DISCUSSION TITLE
How can I make sure my assets are loaded correctly on a subdomain?

### DISCUSSION THREAD

**Discussion by twan-2022** [area17/twill#2426] [Q&A]
I have a site for example website.nl and a twill installation on website.nl/subsite

Now when I go to website.nl/subsite/admin all the css does not load because it tries to get it from website.nl/assets/etc instead of website.nl/subsite/assets/etc.

How can I fix that?

**Comment by ifox** [comment]
Hi @twan-2022, please try setting `TWILL_ASSETS_DIR=subsite/assets/twill` and running `php artisan twill:build`
